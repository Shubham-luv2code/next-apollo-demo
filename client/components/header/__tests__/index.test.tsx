import React from "react";
import Header from "../index";
import renderer from "react-test-renderer";
import { render } from "@testing-library/react";

describe("Header", () => {
  it("should match snapshot", () => {
    const component = renderer.create(<Header text={"This is Header"} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should have header class", () => {
    const { getByText } = render(<Header text={"This is Header"} />);
    const headerName = getByText("This is Header");
    expect(headerName).toHaveClass("header");
  });
});
