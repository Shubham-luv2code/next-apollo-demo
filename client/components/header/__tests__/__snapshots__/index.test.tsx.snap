// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should match snapshot 1`] = `
<div
  className="header"
>
  This is Header
</div>
`;

exports[`Header should match snapshot 1`] = `
<div
  className="header"
>
  This is Header
</div>
`;
